dashboard/app/helpers/course_block_helper.rb

```diff
@@ -106,6 +106,11 @@ def self.get_tall_course_block(id)
         url: CDO.code_org_url('/dance'),
         image_url: CDO.shared_image_url("courses/logo_tall_#{id}.gif")
       },
+      Script::DANCE_PARTY_2019_NAME => {
```

Comment:
Do we still need the original `DANCE_PARTY_NAME` since we're replacing it with DANCE_PARTY_2019_NAME everywhere it's used?

Reply:
Good question.  This particular catalogue is so exhaustive that I figured we'd keep it complete so that we could show a block from any of these tutorials at any time.  Not a strongly-held feeling though.